content/docs/instrumenting/exporters.md

```diff
@@ -119,6 +119,9 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [SNMP exporter](https://github.com/prometheus/snmp_exporter) (**official**)
    * [StatsD exporter](https://github.com/prometheus/statsd_exporter) (**official**)
 
+### Interpreters 
```

Comment:
Will attempt to fix upstream before this gets merged. Leaving as is for now, will kill it in 7 days if there hash been no update.